vermouth/tests/test_ff_files.py

```diff
@@ -513,6 +513,69 @@ def test_param_effector_fail():
         with pytest.raises(IOError):
             vermouth.ffinput.read_ff(lines, ff)
 
+    @staticmethod
+    @pytest.mark.parametrize('interaction_lines, expected', (
+        [
+            """
+            [ dihedrals ]
+            BB SC1 SC2 SC3 2 1 2 3 4
+            BB SC1 SC2 SC4 1
+            """,
+            {'impropers': [vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC3'.split(),
+                                                         parameters='2 1 2 3 4'.split(),
+                                                         meta={})],
+             'dihedrals': [vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC4'.split(),
+                                                         parameters=['1'],
+                                                         meta={})]}
+        ],
+        [
+            """
+            [ dihedrals ]
+            BB SC1 SC2 SC3 1 1 2 3 4
+            BB SC1 SC2 SC4 1
+            """,
+            {'dihedrals': [vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC3'.split(),
+                                                         parameters='1 1 2 3 4'.split(),
+                                                         meta={}), 
+                           vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC4'.split(),
```

Comment:
It's very nitpicking but why use split here? It kind of masks that these are lists of parameters. I am with Jon to have targets as explicit as possible. But I am also fine keeping it like this.